client/src/pages/Login.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+    <v-container no-gutters fluid fill-height class="pa-0 ma-0">
+      <v-row class="d-flex flex-column align-center justify-center">
+        <v-col class="pa-0 ma-0">
+          <v-sheet
+            class="section d-flex align-center"
+            style="padding-bottom: 200px"
+          >
+            <!-- Right side for login component -->
+            <v-col
+              cols="12"
+              xs="12"
+              sm="10"
+              md="10"
+              lg="4"
+              xl="3"
+              xxl="3"
+              class="d-flex flex-column justify-start align-start"
+            >
+                <h1 class="login-text mx-auto">Log In</h1>
+                <div class="text-input-wrapper">
+                    <input class="duolingo-text-input" v-model="email" data-test="email-input" type="text" placeholder="Email or username">
+                    <button @click="clearEmailOrUser" class="cancel-email-input-button _1eJKW _16r-S NF_yB tV2C-" tabindex="-1" type="button">
+                        <img src="https://d35aaqx5ub95lt.cloudfront.net/vendor/8857e57af4c38e2caab2df3d086202da.svg">
+                    </button>
+                </div>
+                <div class="text-input-wrapper">
+                    <input class="duolingo-text-input" data-test="email-input" type="password" placeholder="Password">
+                    <router-link class="forgot-button" tabindex="-1" type="button" to="/forgot-password">
+                        FORGOT?
+                    </router-link>
+                </div>
+                <LAButton text="LOG IN" class="w-100" margin="0px"/>
+                <div class="alt-login-divider">
+                    <div class="horizontal-bar"></div>
+                    <div class="upper-case-text-format or-styling">or</div>
+                    <div class="horizontal-bar"></div>
+                </div>
+                <v-row class="w-100" no-gutters>
+                    <v-col class="mr-2">
+                        <LAButton text="FACEBOOK" class="w-100" margin="0px"/>
+                    </v-col>
+                    <v-col class="ml-2">
+                        <LAButton text="GOOGLE" class="w-100" margin="0px"/>
+                    </v-col>
+                </v-row>
+            </v-col>
+          </v-sheet>
+        </v-col>
+      </v-row>
+    </v-container>
+  </template>
+  
+  <script setup>
+  import { ref } from 'vue'
+  import LAButton from '@/components/LAButton.vue'
+  
+  const email = ref('')
+  const password = ref('')
+  const message = ref('')
+  
+
+  const clearEmailOrUser = () =>  {
+    console.log('Before: ' + email.value)
```

Comment:
Can remove console logs for now. Don't want to forget about any of these.

Reply:
Removed the function